src/log-httplog.c

```diff
@@ -709,6 +709,12 @@ static void LogHttpLogDeInitCtx(OutputCtx *output_ctx)
 {
     LogHttpFileCtx *httplog_ctx = (LogHttpFileCtx *)output_ctx->data;
     uint32_t i;
+
+    if (httplog_ctx->file_ctx->dropped) {
```

Comment:
Could put it in LogFileFreeCtx:
```
[29123] 28/3/2017 -- 19:26:21 - (util-logopenfile.c:639) <Notice> (LogFileFreeCtx) -- 20197 events dropped to socket /tmp/socket
```

I'm thinking a counter in stats would be good as well - perhaps 2 counts. logging.events=<total number of events logged>. logging.dropped=<count> where count is the number of events that weren't delivered, either due to a socket that would block, and socket that was disconnected, or whatever.

Not sure if the complexity of breaking it down into per-logger stats is worth it or not.